tests/browser_test.go

```diff
@@ -343,3 +343,27 @@ func TestMultiConnectToSingleBrowser(t *testing.T) {
 	err = p2.Close(nil)
 	require.NoError(t, err, "failed to close page #2")
 }
+
+func TestCloseContext(t *testing.T) {
+	t.Parallel()
+
+	t.Run("close_context", func(t *testing.T) {
+		t.Parallel()
+
+		tb := newTestBrowser(t)
+		_, err := tb.NewContext(nil)
+		require.NoError(t, err)
+
+		err = tb.CloseContext()
+		require.NoError(t, err)
+
+		assert.Nil(t, tb.Context())
+	})
+
+	t.Run("err_no_context", func(t *testing.T) {
+		t.Parallel()
+
+		tb := newTestBrowser(t)
+		assert.Error(t, tb.CloseContext())
```

Comment:
Also here:

```suggestion
		assert.Nil(t, tb.Context())
		assert.Error(t, tb.CloseContext())
```

Reply:
Fixed in [421c38c](https://github.com/grafana/xk6-browser/pull/1074/commits/421c38c1d373699d98837a6f9a4402391af800d2).